### PR TITLE
Apply path matching/transformation

### PR DESCRIPTION
**Pull request by marcinzaremba**
* applies previously implemented path transformation to the routes
* minor refactoring of func signatures and names

### CODE REVIEW

File: fabric/path.go [hunk 1]
```diff
@@ -26,18 +26,22 @@ var (
...
 	}
 
 	return &eskip.Predicate{
 		Name: predicates.PathName,
-		Args: []interface{}{fp},
+		Args: []interface{}{fps},
 	}
 }
+
+func applyPath(r *eskip.Route, fp *FabricPath) {
```
Comment (szuecs) [inline fabric/path.go:45]: We could also use path as plain string here because we don't use the object.

**Comment by szuecs** [comment]
:+1:

**Comment by marcinzaremba** [comment]
👍

File: fabric/path.go (hunk 1 above)
Comment (szuecs) [reply fabric/path.go:45]: What do you think @marcinzaremba ?

Sorry comment was not showing before now ...

Comment (marcinzaremba) [reply fabric/path.go:45]: Sure, no problem for me. Can change in the next PR.

**Pull request merged**